### ISSUE TITLE
Some namespaces are not prefixed

### ISSUE THREAD

**Issue by Nyholm** [humbug/php-scoper#72]
Example:
```php
<?php

use Symfony\Component\HttpKernel\Kernel;
use Symfony\Component\Config\Loader\LoaderInterface;

class AppKernel extends Kernel
{
    public function registerBundles()
    {
        $bundles = [
            new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
            new \Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
            new AppBundle\AppBundle(),
        ];

        return $bundles;
    }
```

Will be: 

```php
<?php

use PhpScoper595ca8c756b76\Symfony\Component\HttpKernel\Kernel;
use PhpScoper595ca8c756b76\Symfony\Component\Config\Loader\LoaderInterface;
class AppKernel extends Kernel
{
    public function registerBundles()
    {
        $bundles = [
            new Symfony\Bundle\FrameworkBundle\FrameworkBundle(), 
            new \PhpScoper595ca8c756b76\Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(), 
            new AppBundle\AppBundle()
        ];
        return $bundles;
    }
```

**Comment by theofidry** [comment]
This is because `AppKernel` lives in the global namespace, related to https://github.com/humbug/php-scoper/issues/10. With have two ways to go:

- Allow it by providing a config for those functions/classes registered in the global namespace
- Get all the registered constants, functions and classes before anything is autoloaded yet to know what needs to be prefixed or not (was partly attempted in https://github.com/humbug/php-scoper/pull/28)

That said in your case I'm sure namespacing the Kernel to avoid that issue should be trivial

**Comment by Nyholm** [comment]
> That said in your case I'm sure namespacing the Kernel to avoid that issue should be trivial

Yeah. Thank you. That is what I did, I just wanted to report the issue. But since it is a duplicate of #10 I will close this.

**Comment by padraic** [comment]
I'm not sure how complete of an example the above is, but shouldn't `Symfony` and `AppBundle` be prefixed in that array?

**Comment by theofidry** [comment]
@padraic it should be it is not because doesn't start with the leading backslash. Maybe we should account for that as not only it's common in Symfony apps, PHPUnit is doing the same in the `phpunit` script

**Comment by padraic** [comment]
I'm going to reopen the issue so it's not lost, but it should be scoped.

**Comment by theofidry** [comment]
Meh that's a tricky one @padraic, we actually get the same issue as in #10, there is no difference right now between:

```php
new AppKernel();
```

and:

```php
new Closure();
```

That makes me wonder, maybe the initial idea, getting the list of declared classes & co. offers a better experience and may be easier than having to declare the complete list...

**Comment by theofidry** [comment]
Closed as in the end it's a duplicate of #10 which has been fixed :)
